examples/account_create_random.js

```diff
@@ -93,7 +93,7 @@ function delay(ms = 1000) {
 
   const ownerPublicKey = ecc.privateToPublic(process.env.ORE_OWNER_ACCOUNT_KEY);
   const activePublicKey = ecc.privateToPublic(process.env.ORE_OWNER_ACCOUNT_ACTIVE_KEY);
```

Comment:
Can you confirm we no longer need ORE_OWNER_ACCOUNT_ACTIVE_PUBLIC_KEY in the marketplace-web env files? If so, can you remove them?